Laravel-Backpack/CRUD
# Fix PHPDoc of CrudPannel::set

## WHY

### BEFORE - What was wrong? What was happening before this PR?

The PHPDoc of `CrudPannel::set()` is wrong about the 2nd parameter by stating it can be only a boolean value.
So I get PHPStan errors when I use it with a closure for instance.

### AFTER - What is happening after this PR?

No more errors from static analysis tools.

## HOW

### How did you achieve that, in technical terms?

I have edited the PHPDoc.


### Is it a breaking change?

No.


### How can we test the before & after?

Use `CrudPannel::set()` in a project using PHPStan.
Check there is no more errors when the 2nd argument is not a boolean.
